ninjalooter/raidgroups.py

```diff
@@ -0,0 +1,812 @@
+import random
+import math
+
+from models import Player
+
+
+
+
+
+#
+# class for an EQ group
+#
+class Group():
```

Comment:
These will need to be moved to models.py and need to inherit from DictEquals so they have the correct serialization functions (to_json/from_json)